migrate/live-migration/live-migration-from-postgres.md

```diff
@@ -88,43 +88,60 @@ You can do it by pressing the `c` key.
 
 Next, the live migration will migrate the existing data in the source database
 to target database. Then, it will stream live transactions (live replay) received on
-the source side to the target. During this process, it will display the lag between
-the source and target databases in terms of WAL offset size.
+the source side to the target. During this process, it will display the status of the
+replay process.
 
 ```sh
-[WATCH] Source DB - Target DB => 126MB
+Live-replay will complete in 1 minute 38.631 seconds (source_wal_rate: 106.0B/s, target_replay_rate: 589.0KiB/s, replay_lag: 56MiB)
 ```
 
-When the lag between the source and target database is less than 30 megabytes,
-it will start `ANALYZE` on the target database. This updates statistics in the
-target database, which is necessary for optimal querying performance in the
-target database. Wait for `ANALYZE` to complete.
+If the live replay is not able to keep up with the load on the source database, you
+will see a message like this:
+
+```sh
+WARN live-replay not keeping up with source load (source_wal_rate: 3.0MiB/s, target_replay_rate: 462.0KiB/s, replay_lag: 73MiB)
+```
+
+Once the live replay has caught up, the live migration will surface this:
+
+```sh
+Target has caught up with source (source_wal_rate: 751.0B/s, target_replay_rate: 0B/s, replay_lag: 7KiB)
+    To stop replication, hit 'c' and then ENTER
+```
 
 <Highlight type="important">
 Application downtime begins here.
 </Highlight>
 
-Once the lag between the databases is below 30 megabytes, and you're ready to
-take your applications offline, stop all applications which are writing to the
-source database. This is the downtime phase and will last until you have
-completed the validation step (4). Be sure to go through the validation step
-before you enter the downtime phase to keep the overall downtime minimal.
+Once the live replay has caught up, and you're ready to take your applications
```

Comment:
stop all the applications currently writing to the source database. This downtime phase lasts until you [Validate the data in target database and use it as new primary
](#make a link to the step title).

Reply:
done